webdriver_manager/drivers/chrome.py

```diff
@@ -45,6 +45,13 @@ def get_driver_download_url(self):
         if version.parse(driver_version_to_download) < version.parse("106.0.5249.61"):
             os_type = os_type.replace("mac_arm64", "mac64_m1")
 
+        if version.parse(driver_version_to_download) >= version.parse("114.0.5735.00"):
```

Comment:
@david-engelmann this version parse should be >= "115" because that is where the new download url comes into play.

Reply:
@p112913 check out the past action results and this comment https://github.com/SergeyPirogov/webdriver_manager/pull/537#issuecomment-1643671232. 114.0.5735.198 isn't present in the old url

Reply:
@david-engelmann using the new url I get a 404 for this version:
https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/114.0.5735.198/mac-arm64/chromedriver-mac-arm64.zip

Also the last known version I see on release 114 is 114.0.5735.90 and even for that the new url doesn't work. You can try this in your browser and you would get back a 404:

Reply:
gotcha so it looks like the testing pipelines has a bad test for 114.0.5735.1988. I'll update the version check

Reply:
Failed test in question
```
FAILED tests/test_brave_driver.py::test_brave_manager_with_selenium - ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/115/chromedriver_linux64.zip
FAILED tests/test_chrome_driver.py::test_chrome_manager_with_cache - ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/114.0.5735.198/chromedriver_linux64.zip
FAILED tests/test_chrome_driver.py::test_chrome_manager_with_selenium - ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/114.0.5735.198/chromedriver_linux64.zip
FAILED tests/test_chrome_driver.py::test_driver_with_ssl_verify_disabled_can_be_downloaded - ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/114.0.5735.198/chromedriver_linux64.zip
FAILED tests/test_chrome_driver.py::test_chrome_manager_cached_driver_with_selenium - ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/114.0.5735.198/chromedriver_linux64.zip
FAILED tests/test_chrome_driver.py::test_can_get_chrome_for_win[win32] - ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/114.0.5735.198/chromedriver_win32.zip
FAILED tests/test_chrome_driver.py::test_can_get_chrome_for_win[win64] - ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/114.0.5735.198/chromedriver_win32.zip
FAILED tests/test_chrome_driver.py::test_can_get_chrome_for_win[mac64] - ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/114.0.5735.198/chromedriver_mac_arm64.zip
FAILED tests/test_chrome_driver.py::test_can_get_chrome_for_win[mac64_m1] - ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/114.0.5735.198/chromedriver_mac_arm64.zip
```

Reply:
I see the last known 114.0.5735.90 available for download. The filename was different then the ones you tried. I'm still ok with starting with version 115 for the new endpoint. One thing to consider, people used to pass versions="114" etc. I'm not sure that format works with the new endpoint

here is the json block 
```
{
      "version": "114.0.5735.90",
      "revision": "1135570",
      "downloads": {
        "chrome": [
          {
            "platform": "linux64",
            "url": "https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/114.0.5735.90/linux64/chrome-linux64.zip"
          },
          {
            "platform": "mac-arm64",
            "url": "https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/114.0.5735.90/mac-arm64/chrome-mac-arm64.zip"
          },
          {
            "platform": "mac-x64",
            "url": "https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/114.0.5735.90/mac-x64/chrome-mac-x64.zip"
          },
          {
            "platform": "win32",
            "url": "https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/114.0.5735.90/win32/chrome-win32.zip"
          },
          {
            "platform": "win64",
            "url": "https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/114.0.5735.90/win64/chrome-win64.zip"
          }
        ]
      }
    },
```

Reply:
oh, so I was missing the "chromedriver" at the end in my url. I think since versions below 115 are still accessible on the old url so let's use the old url for all versions below 115 and then starting >= 115 we can use the new url.

Reply:
for passing of version I don't know how it will supported by the new url format. I am not sure of what the logic is for versions 114 and below but I think it we should retain that feature perhaps with versions >=115 ? I can see some more work probably needs to be put in to have that feature supported with the new url

Reply:
we can use https://googlechromelabs.github.io/chrome-for-testing/last-known-good-versions.json similarly to the LATEST_RELEASE endpoint of old

Reply:
Hi, ChromeDriver lead here. The new locations only apply to 115 and later. For 114 and below the old URLs still apply.

Reply:
@mathiasbynens Perfect, I've just added support for the simple version ie. 115 and the build version ie. 115.0.5735